src/services/store.ts

```diff
@@ -52,8 +52,8 @@ export function addToFavoriteLabelValueInStorage(dsKey: string | unknown, labelN
   localStorage.setItem(key, JSON.stringify(servicesToStore));
 }
 
-export function removeFromFavoritesInStorage(dsKey: string, labelName: string, labelValue: string) {
-  if (!dsKey || !labelName || !labelValue) {
+export function removeFromFavoritesInStorage(dsKey: string | unknown, labelName: string, labelValue: string) {
```

Comment:
Is it unknown or undefined? Seems unexpected that it could be unknown.

Reply:
yeah good catch, I was looking at the other functions in the store that do this, but it really should be a `VariableValue` instead (updated)